pkg/asset/ignition/bootstrap/common.go

```diff
@@ -709,12 +709,12 @@ func (a *Common) load(f asset.FileFetcher, filename string) (found bool, err err
 	}
 
 	a.File, a.Config = file, config
-	warnIfCertificatesExpired(a.Config)
-	return true, nil
+	err = warnIfCertificatesExpired(a.Config)
+	return true, err
 }
 
 // warnIfCertificatesExpired checks for expired certificates and warns if so
-func warnIfCertificatesExpired(config *igntypes.Config) {
+func warnIfCertificatesExpired(config *igntypes.Config) error {
```

Comment:
note: sorry, I can't pin the line here but it is with:

```
logrus.Warnf("Bootstrap Ignition-Config Certificate %s expired at %s.", path.Base(file.Path), cert.NotAfter.Format(time.RFC3339))
```

Since we now treat expired certs as an error, should we now promote the log level to error (i.e. `logrus.Errorf`?

Reply:
we can do that I think its splitting hairs. I will make the change but it may actually be more confusing. Now you would see say 10 error message then the error saying 10 certs failed.